lab_1_classify_by_unigrams/start.py

```diff
@@ -2,14 +2,15 @@
 Language detection starter
 """
 # pylint:disable=too-many-locals, unused-argument, unused-variable
-
+import main as m
 
 def main() -> None:
     """
     Launches an implementation
     """
     with open("assets/texts/en.txt", "r", encoding="utf-8") as file_to_read_en:
         en_text = file_to_read_en.read()
+        m.create_language_profile('en', en_text)
```

Comment:
can you show the full work of your code here?

Reply:
fixed